packages/components/data-resource/s3-endpoint-type.tsx

```diff
@@ -30,8 +35,17 @@ type S3EndpointTypeProps = {
   namespace: string;
 };
 
+const { namespace, ...secretResourceWithoutNamespace } = secretResource;
+const updatedSecretResource: WatchK8sResource = {
+  ...secretResourceWithoutNamespace,
+  namespaced: false,
+};
+
 const regionDropdownOptions: (t: TFunction) => JSX.Element[] = (t) =>
   _.map(awsRegionItems, (v, _) => <SelectOption key={v} value={v} />);
+const getSecondaryTextForSecrets = (resource): string => {
+  return (resource as K8sResourceCommon)?.metadata?.namespace;
+};
```

Comment:
```suggestion
const getSecondaryTextForSecrets = (resource: K8sResourceCommon): string => {
  return resource?.metadata?.namespace;
};
```
it will be auto converted